ipn/state_enum_encoding.go

```diff
@@ -0,0 +1,56 @@
+// Code generated by go-enum-encoding DO NOT EDIT
```

Comment:
This is missing the final period from the prescribed pattern (go help generate)

It should be `^// Code generated .* DO NOT EDIT\.$`

And conventionally there's a semicolon before "; DO NOT EDIT."

Reply:
good point. fixing this.